packages/grafana-ui/src/components/FilterPill/FilterPill.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { FilterPill } from './FilterPill';
+
+const onClick = jest.fn();
+
+const setup = (jsx: JSX.Element) => {
+  return {
+    user: userEvent.setup(),
+    ...render(jsx),
+  };
+};
```

Comment:
Is this something that we need? I don't recall having to do this in other tests 🤔 

I think userEvent.setup() is useful to specify config like the timing of the events, but otherwise I don't think it's necessary? If we can do without it, and the setup function, it would be great!